rust/src/accuracy/tail_bounds/conservative_continuous_gaussian_tail_to_alpha.tex

```diff
@@ -0,0 +1,63 @@
+\documentclass{article}
+\input{../../lib.sty}
+
+\title{\texttt{fn conservative\_continuous\_gaussian\_tail\_to\_alpha}}
+\author{Michael Shoemate}
+
+\begin{document}
+\maketitle
+
+\contrib
+
+Proof for \rustdoc{accuracy/tail\_bounds/fn}{conservative\_continuous\_gaussian\_tail\_to\_alpha}.
+
+\begin{definition}
+    \label{gaussian}
+    Define $X \sim \mathcal{N}(0, s)$, a random variable following the continuous gaussian distribution:
+    \begin{equation}
+        f(x) = \frac{1}{\sigma \sqrt{2 \pi}} e^{-\frac{1}{2}\left( \frac{x}{\sigma}\right)^2}
+    \end{equation}
+\end{definition}
+
+\begin{definition}
+    \label{erf}
+    The error function is defined as:
+    \begin{equation}
+        \mathrm{erf}(z) = \frac{2}{\sqrt{\pi}} \int_{0}^{z} e^{-t^2} dt
+    \end{equation}
+\end{definition}
+
+\begin{definition}
+    \label{erfc}
+    The complementary error function is defined as:
+    \begin{equation}
+        \mathrm{erfc}(z) = 1 - \mathrm{erf}(z)
+    \end{equation}
+\end{definition}
+
+\begin{lemma}
+    \label{erfc-err}
+    By exhaustive search in \texttt{erfc\_err\_analysis.py}, the implementation of \texttt{erfc} errs by no greater than one 32-bit float ulp.
```

Comment:
The script is pretty short: Could we include the code in the rendered PDF, or make this a link to the file in github? _and_ run it in CI on some periodic basis?